ledger/participant-state-errors/src/main/scala/com/daml/ledger/participant/state/kv/errors/KVErrors.scala

```diff
@@ -15,6 +13,8 @@ import com.daml.error.{
   Resolution,
 }
 
+import java.time.Instant
+
 @Explanation(
```

Comment:
@pbatko-da, is the documentation going to be generated after this move? How to check it?